client/components/QuestionsAndAnswers/modal/AnswerModal.jsx

```diff
@@ -1,9 +1,12 @@
 import React, { useState } from 'react';
 import ReactDOM from 'react-dom';
-import { ModalShadow, Modal } from '../styles/Modal.style';
 import AnswerForm from './AnswerForm';
+import { useProduct } from '../../../ProductContext';
+import { ModalShadow, Modal } from '../styles/Modal.style';
 
 const AnswerModal = ({ question, setIsAdd, handleAddAnswer }) => {
+  const currProduct = useProduct();
+  console.log('what is currP', currProduct);
   const [newAnswer, setNewAnswer] = useState({
```

Comment:
I think if you had an onSubmit in the form you wouldn't need to pass down this object & you can handle submit from the form component directly instead of passing state back and forth through both components.